docs/reference/rollup/overview.asciidoc

```diff
@@ -23,7 +26,7 @@ reading often diminishes with time.  It's not useless -- it could easily contrib
 value often leads to deletion rather than paying the fixed storage cost.
 
 [float]
-=== Rollup store historical data at reduced granularity
+==== Rollup store historical data at reduced granularity
 
```

Comment:
"Rollup store historical data" is clunky. Rollup stores? Roll up to store? Or maybe "Storing historical data at reduced granularity"? (The pattern here of repeating Rollup in every heading kind seems like overkill.)

Reply:
I've updated it to match the other sections, but I agree this page could do with an edit. Will defer and just stick to structural changes here, however.